test/martha/martha_v3.test.js

```diff
@@ -653,15 +621,14 @@ test.serial('martha_v3 parses a Gen3 CRDC CIB URI response correctly', async (t)
     t.is(actualSAKeyProvider, expectedSAKeyProvider);
 });
 
+// BT-236 temporarily cut access token and access endpoint out of the flow
 test.serial('martha_v3 parses BDC response correctly', async (t) => {
     const drsAccessUrlResponse = mockGcsAccessUrl(bdcDrsResponse.access_methods[0].access_url.url);
     // Rotated once from Michael Baumann's sequence diagram so that call 0 is the second call down from the top,
     // call 3 is the top call.
     // https://lucid.app/lucidchart/428a0bdd-a884-4fc7-9a49-7bf300ef6777/edit?shared=true&page=0_0#
     getJsonFromApiStub.onCall(0).resolves(bdcDrsResponse);
-    getJsonFromApiStub.onCall(1).resolves(bondAccessTokenResponse);
-    getJsonFromApiStub.onCall(2).resolves(drsAccessUrlResponse);
-    getJsonFromApiStub.onCall(3).resolves(googleSAKeyObject);
+    getJsonFromApiStub.onCall(1).resolves(googleSAKeyObject);
```

Comment:
FYI: There's no more "call 3", and I just noticed the sequence diagram doesn't have a link.

Reply:
that link appears to be working for me...

Reply:
🤦